common-init.mk

```diff
@@ -21,6 +21,7 @@ PRODUCT_PACKAGES += \
     cdsprpcd.rc \
     cnss-daemon.rc \
     ipacm.rc \
+    dpmQmiMgr.rc \
```

Comment:
Can the sm8150-specific stuff be guarded by TARGET_PRODUCT?

Also, for later, maybe guard the IMS stuff by `TARGET_HAS_IMS`.

Reply:
Not a problem to have it on legacy targets, AFAIK.